src/models/mods.rs

```diff
@@ -96,15 +96,15 @@ pub struct FileHash {
     pub hash: String,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, Clone)]
 pub enum VersionType {
```

Comment:
Making this a simple string that is possible to edit should be better. It allows for more channels and avoids the conflicts of release settings

Reply:
We could use a string, but we would still want a limited list of allowed version types - we don't want to have to deal with people labeling their versions with inappropriate language, or labeling their versions with types that make no sense to most people.